diff -ruN a/app/common_dialog.js b/app/common_dialog.js
--- a/app/common_dialog.js
+++ b/app/common_dialog.js
@@ -31,7 +31,10 @@
     },
     58933: (e) => {
       "use strict";
-      e.exports = require("electron");
+      e.exports = {
+        ...require("electron"),
+        remote: require("@electron/remote"),
+      };
     },
     28614: (e) => {
       "use strict";
diff -ruN a/app/download_progress.js b/app/download_progress.js
--- a/app/download_progress.js
+++ b/app/download_progress.js
@@ -73,7 +73,10 @@
     },
     58933: (_) => {
       "use strict";
-      _.exports = require("electron");
+      _.exports = {
+        ...require("electron"),
+        remote: require("@electron/remote"),
+      };
     },
     12087: (_) => {
       "use strict";
diff -ruN a/app/index.js b/app/index.js
--- a/app/index.js
+++ b/app/index.js
@@ -753,7 +753,10 @@
     },
     58933: (n) => {
       "use strict";
-      n.exports = require("electron");
+      n.exports = {
+        ...require("electron"),
+        remote: require("@electron/remote"),
+      };
     },
     28614: (n) => {
       "use strict";
diff -ruN a/app/login.js b/app/login.js
--- a/app/login.js
+++ b/app/login.js
@@ -17,7 +17,10 @@
     },
     58933: (e) => {
       "use strict";
-      e.exports = require("electron");
+      e.exports = {
+        ...require("electron"),
+        remote: require("@electron/remote"),
+      };
     },
     28614: (e) => {
       "use strict";
diff -ruN a/app/lyric.js b/app/lyric.js
--- a/app/lyric.js
+++ b/app/lyric.js
@@ -301,7 +301,10 @@
     },
     58933: (_) => {
       "use strict";
-      _.exports = require("electron");
+      _.exports = {
+        ...require("electron"),
+        remote: require("@electron/remote"),
+      };
     },
     28614: (_) => {
       "use strict";
diff -ruN a/app/main.js b/app/main.js
--- a/app/main.js
+++ b/app/main.js
@@ -1,3 +1,8 @@
+const { app: __remoteApp } = require("electron");
+require("@electron/remote/main").initialize();
+__remoteApp.on("browser-window-created", (_event, window) =>
+  require("@electron/remote/main").enable(window.webContents),
+);
 /*! For license information please see main.js.LICENSE.txt */
 (() => {
   var e = {
@@ -8899,6 +8904,7 @@
           frame: !1,
           show: !1,
           webPreferences: {
+            contextIsolation: !1,
             webviewTag: !0,
             webSecurity: !1,
             enableRemoteModule: !0,
@@ -9884,6 +9890,7 @@
                         secure: !0,
                         path: "/",
                         domain: "qq.com",
+                        sameSite: "no_restriction",
                       })
                     : r.remote.session.defaultSession.cookies.set({
                         url: "https://u.y.qq.com/cgi-bin/musics.fcg",
@@ -9892,6 +9899,7 @@
                         secure: !0,
                         path: "/",
                         domain: "qq.com",
+                        sameSite: "no_restriction",
                       });
                 }),
               )
@@ -9943,6 +9951,7 @@
               width: 540,
               height: 440,
               webPreferences: {
+                contextIsolation: !1,
                 webviewTag: !0,
                 webSecurity: !1,
                 enableRemoteModule: !0,
diff -ruN a/app/visible_play.js b/app/visible_play.js
--- a/app/visible_play.js
+++ b/app/visible_play.js
@@ -23,7 +23,10 @@
     },
     58933: (e) => {
       "use strict";
-      e.exports = require("electron");
+      e.exports = {
+        ...require("electron"),
+        remote: require("@electron/remote"),
+      };
     },
   },
   (e) => {
diff -ruN a/modules/876/69458.js b/modules/876/69458.js
--- a/modules/876/69458.js
+++ b/modules/876/69458.js
@@ -44,6 +44,8 @@
   return /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0__.createElement(react__WEBPACK_IMPORTED_MODULE_0__.Fragment, null, (loading || keepAliveDelayLoading) && /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0__.createElement(_src_component_loading_qqmusic_loading__WEBPACK_IMPORTED_MODULE_2__/* .QQMusicLoading */ .Z, {
     className: "loading_cover"
   }), /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0__.createElement("webview", {
+    nodeintegration: '',
+    webpreferences: 'contextIsolation=false',
     preload: _src_lib_common_utils__WEBPACK_IMPORTED_MODULE_1__/* .default.returnPreload */ .ZP.returnPreload(),
     src: src,
     id: id,
diff -ruN a/modules/common/68010.js b/modules/common/68010.js
--- a/modules/common/68010.js
+++ b/modules/common/68010.js
@@ -335,7 +335,8 @@
           value: item.value,
           secure: true,
           path: '/',
-          domain: 'qq.com'
+          domain: 'qq.com',
+          sameSite: 'no_restriction'
         });
       } else {
         electron__WEBPACK_IMPORTED_MODULE_3__.remote.session.defaultSession.cookies.set({
@@ -344,7 +345,8 @@
           value: item.value,
           secure: true,
           path: '/',
-          domain: 'qq.com'
+          domain: 'qq.com',
+          sameSite: 'no_restriction'
         });
       }
     })).then(() => {
@@ -405,6 +407,7 @@
       width: 540,
       height: 440,
       webPreferences: {
+        contextIsolation: false,
         webviewTag: true,
         webSecurity: false,
         enableRemoteModule: true,
diff -ruN a/modules/login/70313.js b/modules/login/70313.js
--- a/modules/login/70313.js
+++ b/modules/login/70313.js
@@ -70,6 +70,8 @@
         height: '100%'
       }
     }, /*#__PURE__*/react.createElement("webview", {
+    nodeintegration: '',
+    webpreferences: 'contextIsolation=false',
       src: "https://y.qq.com/wk_v17/common_login.html#/login?WXAppId=wx1a376b527b64b71d",
       style: {
         width: '100%',
