diff -ruN a/app/main.js b/app/main.js
--- a/app/main.js
+++ b/app/main.js
@@ -9606,7 +9606,7 @@
               (e.comm.tmeAppID = "qqmusic"))
             : (e.comm = { ct: 20, cv: 1770, tmeAppID: "qqmusic" });
           const r = Object.keys(e).some((e) => Object.values(de).includes(e));
-          if ((console.log("params", e, r), t || r)) {
+          if (t || r) {
             a = "https://u.y.qq.com/cgi-bin/musics.fcg";
             const t = he(e, !0);
             a = me({ sign: fe(t) }, a);
@@ -9678,7 +9678,7 @@
               .then((e) =>
                 0 === e.code && e.login && 0 === e.login.code && e.login.data
                   ? this.handleWechatLoginData(e.login.data)
-                  : (console.error("WX login by token fail", e),
+                  : (console.error("WX login by token fail"),
                     Promise.reject()),
               )
               .catch(
@@ -9705,7 +9705,7 @@
                   (null === (t = e.login) || void 0 === t ? void 0 : t.code) &&
                 e.login.data
                 ? this.handleWechatLoginData(e.login.data)
-                : (console.error("QQ login by token fail", e),
+                : (console.error("QQ login by token fail"),
                   Promise.reject());
             });
           }
@@ -9739,7 +9739,7 @@
                       : t.code) &&
                   e.login.data
                   ? this.handleQQLoginData(e.login.data)
-                  : (console.error("QQ login by token fail", e),
+                  : (console.error("QQ login by token fail"),
                     Promise.reject());
               })
               .catch(
@@ -9766,7 +9766,7 @@
                   (null === (t = e.login) || void 0 === t ? void 0 : t.code) &&
                 e.login.data
                 ? this.handleQQLoginData(e.login.data)
-                : (console.error("QQ login by token fail", e),
+                : (console.error("QQ login by token fail"),
                   Promise.reject());
             });
           }
@@ -9907,7 +9907,7 @@
                   console.log("cookie set success");
                 })
                 .catch((e) => {
-                  (console.log("cookie set fail"), console.log(e));
+                  console.log("cookie set fail");
                 })
             );
           }
diff -ruN a/modules/common/32590.js b/modules/common/32590.js
--- a/modules/common/32590.js
+++ b/modules/common/32590.js
@@ -109,7 +109,6 @@
 
 
   const shouldEncrypt = Object.keys(params).some(key => Object.values(SECURITY_METHODS).includes(key));
-  console.log('params', params, shouldEncrypt); // cgi-bin/musics.fcg 增加加密签名
 
   if (isSecurityCGI || shouldEncrypt) {
     url = 'https://u.y.qq.com/cgi-bin/musics.fcg';
diff -ruN a/modules/common/68010.js b/modules/common/68010.js
--- a/modules/common/68010.js
+++ b/modules/common/68010.js
@@ -76,7 +76,7 @@
       if (res.code === 0 && res.login && res.login.code === 0 && res.login.data) {
         return this.handleWechatLoginData(res.login.data);
       } else {
-        console.error('WX login by token fail', res);
+        console.error('WX login by token fail');
         return Promise.reject();
       }
     }).catch(() => {
@@ -102,7 +102,7 @@
       if (res.code === 0 && res.login && ((_res$login = res.login) === null || _res$login === void 0 ? void 0 : _res$login.code) === 0 && res.login.data) {
         return this.handleWechatLoginData(res.login.data);
       } else {
-        console.error('QQ login by token fail', res);
+        console.error('QQ login by token fail');
         return Promise.reject();
       }
     });
@@ -136,7 +136,7 @@
       if (res.code === 0 && res.login && ((_res$login2 = res.login) === null || _res$login2 === void 0 ? void 0 : _res$login2.code) === 0 && res.login.data) {
         return this.handleQQLoginData(res.login.data);
       } else {
-        console.error('QQ login by token fail', res);
+        console.error('QQ login by token fail');
         return Promise.reject();
       }
     }).catch(() => {
@@ -163,7 +163,7 @@
       if (res.code === 0 && res.login && ((_res$login3 = res.login) === null || _res$login3 === void 0 ? void 0 : _res$login3.code) === 0 && res.login.data) {
         return this.handleQQLoginData(res.login.data);
       } else {
-        console.error('QQ login by token fail', res);
+        console.error('QQ login by token fail');
         return Promise.reject();
       }
     });
@@ -351,9 +351,8 @@
       }
     })).then(() => {
       console.log('cookie set success');
-    }).catch(error => {
+    }).catch(() => {
       console.log('cookie set fail');
-      console.log(error);
     });
   }
 
diff -ruN a/modules/login/70313.js b/modules/login/70313.js
--- a/modules/login/70313.js
+++ b/modules/login/70313.js
@@ -30,7 +30,6 @@
 
         if (args.cmd == 'clientLogin') {
           if (args.params.type == 'QQ') {
-            console.log('code', args.params);
             login/* default.qqLoginByCode */.Z.qqLoginByCode(args.params.code).then(() => {
               console.log('QQ登录成功');
               this.notifyMainWindow();
